infra/packer/Dockerfile.build

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:focal-20211006
+
+# Install system dependencies
+RUN apt update \
+  && apt install -y --no-install-recommends \
+    ca-certificates \
+    curl \
+    gcc \
+    git \
+    libc6-dev \
+    make \
+  && apt clean autoclean \
+  && apt autoremove --yes \
+  && rm -rf /var/lib/{apt,dpkg,cache,log}/
+
+# Install go 
+RUN curl -sL https://go.dev/dl/go1.17.5.linux-amd64.tar.gz | tar --strip-components=1 -C /usr/local -xz
```

Comment:
Would there be any value in getting this to sync with the go version on line 3 of `go.mod`?

Reply:
I could do that, it only gives us  the major version though (1.17). Could always float to latest minor on that major?

Reply:
May as well leave as-is then